help/add-or-remove-users-from-a-stream.md

```diff
@@ -90,6 +90,29 @@ This method is useful if you need to remove one user from multiple streams.
 
 {end_tabs}
 
+## Configure who can remove users
+
+{!admin-only.md!}
+
+Organization administrators can configure which user roles can remove other
```

Comment:
In the future, we expect this to be configurable with groups rather than roles. How about writing this as, "configure who can remove..." so that we don't have to fix it later?